lib/tests/common.go

```diff
@@ -0,0 +1,144 @@
+package test
+
+import (
+	"archive/tar"
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/appc/docker2aci/lib/internal/typesV2"
+)
+
+type Layer map[*tar.Header][]byte
```

Comment:
When I make that change I get:

```
derek@proton ~/go/src/github.com/appc/docker2aci> go test -v github.com/appc/docker2aci/lib/tests
# github.com/appc/docker2aci/lib/tests                                                          
lib/tests/common.go:16: invalid map key type tar.Header
```

No clue why go doesn't allow this, but I don't think that using pointers will be an issue here. This is only used in the creation of tests, and test writers can avoid creating duplicate `tar.Header`s.